services/registration/src/routes/application.ts

```diff
@@ -23,6 +56,9 @@ applicationRouter.route("/:id").get(
 
 applicationRouter.route("/").post(
   asyncHandler(async (req, res) => {
+    if (!validateApplicationData(req.body.applicationData)) {
+      return res.status(400).send("Failure validating application data");
+    }
```

Comment:
For this, and below, lets use `BadRequestError` you can search the rest of the code base for examples. Additionally, could you modify the code to return the specific validation error instead of just "Failure validating application data"?